pkgs/tools/compression/lrzip/default.nix

```diff
@@ -1,17 +1,33 @@
-{ lib, stdenv, fetchurl, zlib, lzo, bzip2, lz4, nasm, perl }:
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, zlib, lzo, bzip2, lz4, nasm, perl }:
 
+let
+  isx86 = stdenv.isx86_64 || stdenv.isi686;
```

Comment:
```suggestion
  inherit (stdenv.hostPlatform) isx86;
```

Reply:
Thanks. Updated.

There are a few other places with the same pattern:

``` sh
$ rg 'stdenv\.(hostPlatform\.)?isx86_64.*stdenv\.(hostPlatform\.)?isi686'
pkgs/os-specific/linux/firmware/fwupd/default.nix
61:  isx86 = stdenv.isx86_64 || stdenv.isi686;

pkgs/development/libraries/x264/default.nix
40:  nativeBuildInputs = lib.optional (stdenv.hostPlatform.isx86_64 || stdenv.hostPlatform.isi686) nasm;

pkgs/development/libraries/glibc/common.nix
164:    ] ++ lib.optionals (stdenv.hostPlatform.isx86_64 || stdenv.hostPlatform.isi686 || stdenv.hostPlatform.isAarch64) [

pkgs/desktops/gnome/core/gnome-software/default.nix
40:  withFwupd = stdenv.isx86_64 || stdenv.isi686;

pkgs/tools/networking/fastd/default.nix
45:  mesonFlags = lib.optionals (!stdenv.isx86_64 && !stdenv.isi686) [
```

Reply:
I think `isx86` is not present under `stdenv` at the top-level, and so many people don't know it exists.

Reply:
I opened a PR to hopefully address this here: https://github.com/NixOS/nixpkgs/pull/146829